lib/transports/sshTransport.js

```diff
@@ -15,11 +15,9 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-function sshCall(config, xmlIn, done) {
-  // ssh2 is an optional dependency, since users may not use this transport
-  // eslint-disable-next-line global-require, import/no-extraneous-dependencies
-  const { Client } = require('ssh2');
+const { Client } = require('ssh2');
```

Comment:
This module written entirely in JS so are its deps. This package should always install regardless of the platform.